pilot/pkg/proxy/envoy/v2/cds.go

```diff
@@ -92,3 +92,46 @@ func (s *DiscoveryServer) generateRawClusters(node *model.Proxy, push *model.Pus
 	}
 	return rawClusters, nil
 }
+
+// Set the SDS token path if SDS_TOKEN_PATH is defined in the proxy metadata
+func SetSdsTokenPathFromProxyMetadata(c *xdsapi.Cluster, node *model.Proxy) {
```

Comment:
Where do we actually use this function?

Reply:
Fixed: this function is called in generateRawClusters().

Reply:
Thanks for working on this!

I feel sds token is not an accurate name and may cause confusion. Since this is the call credential for SDS request, not "SDS's" token. 

How about calling it something like ...TokenPathForSds... ?

Reply:
And please update the comments accordingly, thanks.

Reply:
Done: changed to ...TokenPathForSds... and changed the comment accordingly.